Makefile

```diff
@@ -26,14 +26,7 @@ generate:
 .PHONY: ci
 ci: check-tidy test e2e-test
 
-.PHONY: start-emulator
-start-emulator:
-	./flow-x86_64-linux- emulator --evm-enabled
-
-.PHONY: setup-account
-setup-account:
-	./flow-x86_64-linux- transactions send api/cadence/transactions/create_bridged_account.cdc 1500.0 --network=emulator --signer=emulator-account
-
 .PHONY: start
 start:
-	go run ./cmd/server/main.go
\ No newline at end of file
+	rm -rf db/
+	go run cmd/main/main.go --flow-network-id=flow-emulator --coinbase=FACF71692421039876a5BB4F10EF7A439D8ef61E --coa-address=f8d6e0586b0a20c7 --coa-key=2619878f0e2ff438d17835c2a4561cb87b4d24d72d12ec34569acd0dd4af7c21 --coa-resource-create=true --gas-price=0
```

Comment:
I wouldn't set this as the `start` command. I would leave the start command as it is and maybe set this as `start-local` or something else, because it could be misused

Reply:
Good point, updated in https://github.com/onflow/flow-evm-gateway/pull/132/commits/50a8024bc8c21b5a3859311d407d272e2b3ee21e